README.md

```diff
@@ -18,8 +18,8 @@ Action requires the following parameters:
    * `api-key-secret` - APS api key secret
    * `app_file` -  mobile application file (_.zip_, _.apk_ or _aab_)
 
-APS key id and APS key secret can be created and retrieved in [Verimatrix App Shield](https://appshield.verimatrixcloud.net/Developer)
-portal under **My Account -> Developer** menu.  
+APS key id and APS key secret can be created and retrieved in [Verimatrix XTD and Counterspy](https://portal.platform.verimatrixcloud.net/)
+portal under **Settings -> API Key Manager** menu.  
```

Comment:
In fact, that "API Key Manager" menu is bound to confuse a few million people 😄 ... unfortunately, that's not where the CI/CD keys are (they will be, but not yet).
For the time being, theyre under the Overview menu, in the "CI/CD integration" panel, "Manage API Keys"